packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/base-types.ts

```diff
@@ -500,6 +500,28 @@ export enum CompressionType {
 // Create Transform Job types
 //
 
+/**
+ * Configures the timeout and maximum number of retries for processing a transform job invocation.
+ *
+ *  @experimental
+ */
+export interface ModelClientOptions {
+
+  /**
+   * The maximum number of retries when invocation requests are failing.
+   *
+   * @default 0
+   */
+  readonly invocationsMaxRetries?: number;
```

Comment:
should these properties be named in the singular (invocation* rather than invocations*)

Reply:
I think invocations makes more sense because:
1. That is what the API uses.
2. Batch transform is not a single invocation but for a whole batch (more than 1 query, and typically more than 1 input file).

Reply:
I agree, let's leave it as it is. thanks for the clarification!